tmt/steps/__init__.py

```diff
@@ -916,10 +919,12 @@ def _emit_key(key: str) -> None:
 
     def enabled_on_guest(self, guest: 'Guest') -> bool:
         """ Check if the plugin is enabled on the specific guest """
-        where: str = self.get('where')
+        where = cast(List[str], self.get('where'))
```

Comment:
Are we planning to remove the need of this cast, or is it here to stay?

I see that `def get` of `BasePlugin` returns `Any`, but it looks up the info from `self.data` what is a dataclass?
It would be nice if it would not be needed ...

Reply:
(Added a comment d6a9e737).

That's the correct observation - in most cases, I believe `get()` can be replaced with simple `self.data` access. But, there are some plugins that use special defaults, which need to be taken care of before we can get rid of `get()`.